src/main/java/uk/gov/ons/census/notifyprocessor/service/FulfilmentRequestService.java

```diff
@@ -15,6 +18,18 @@
 @Service
 public class FulfilmentRequestService {
 
+  private static final String INDIVIDUAL_QUESTIONNAIRE_REQUEST_ENGLAND = "UACIT1";
```

Comment:
Being pedantic here, but in case-processor these are called `HOUSEHOLD_INDIVIDUAL_RESPONSE_REQUEST_<COUNTRY>_<OPTIONAL_LANGUAGE>_SMS`. Would these need to match?